qiskit_nature/runtime/vqe_program.py

```diff
@@ -41,342 +47,52 @@ def __init__(
-                )
-
-            _validate_optimizer_settings(optimizer)
-
-            self._optimizer = optimizer
-
-    @property
-    def backend(self) -> Optional[Backend]:
-        """Returns the backend."""
-        return self._backend
-
-    @backend.setter
-    def backend(self, backend) -> None:
-        """Sets the backend."""
-        self._backend = backend
-
-    @property
-    def initial_point(self) -> Optional[np.ndarray]:
-        """Returns the initial point."""
-        return self._initial_point
-
-    @initial_point.setter
-    def initial_point(self, initial_point: Optional[np.ndarray]) -> None:
-        """Sets the initial point."""
-        self._initial_point = initial_point
-
-    @property
-    def shots(self) -> int:
-        """Return the number of shots."""
-        return self._shots
-
-    @shots.setter
-    def shots(self, shots: int) -> None:
-        """Set the number of shots."""
-        self._shots = shots
-
-    @property
-    def measurement_error_mitigation(self) -> bool:
-        """Returns whether or not to use measurement error mitigation.
-
-        Readout error mitigation is done using a complete measurement fitter with the
-        ``self.shots`` number of shots and re-calibrations every 30 minutes.
-        """
-        return self._measurement_error_mitigation
-
-    @measurement_error_mitigation.setter
-    def measurement_error_mitigation(self, measurement_error_mitigation: bool) -> None:
-        """Whether or not to use readout error mitigation."""
-        self._measurement_error_mitigation = measurement_error_mitigation
-
-    @property
-    def store_intermediate(self) -> bool:
-        """Returns whether or not to store intermediate information of the optimization."""
-        return self._store_intermediate
-
-    @store_intermediate.setter
-    def store_intermediate(self, store: bool) -> None:
-        """Whether or not to store intermediate information of the optimization."""
-        self._store_intermediate = store
-
-    @property
-    def callback(self) -> Callable:
-        """Returns the callback."""
-        return self._callback
-
-    @callback.setter
-    def callback(self, callback: Callable) -> None:
-        """Set the callback."""
-        self._callback = callback
-
-    def _wrap_vqe_callback(self) -> Optional[Callable]:
-        """Wraps and returns the given callback to match the signature of the runtime callback."""
-
-        def wrapped_callback(*args):
-            _, data = args  # first element is the job id
-            iteration_count = data[0]
-            params = data[1]
-            mean = data[2]
-            sigma = data[3]
-            return self._callback(iteration_count, params, mean, sigma)
-
-        # if callback is set, return wrapped callback, else return None
-        if self._callback:
-            return wrapped_callback
-        else:
-            return None
+        warn_deprecated(
+            version="0.2.1",
```

Comment:
```suggestion
            version="0.3.0",
```
I think this is deprecated as of 0.3.0.